docs/faq.md

```diff
@@ -0,0 +1,77 @@
+
+# Tool Kit FAQs
+
+Below are responses to some general questions about Tool Kit that have been asked during migrations. If your question is not answered here please ask in the [#tool-kit](https://app.slack.com/client/T025C95MN/C02TRE2V2Q1/thread/C042NBBTM-1671107986.307219) slack channel and consider adding it, and the answer, to this document.
+
+### How do I add package specific configuration (for e.g. ESLint) to the Tool Kit plugin (e.g. @dotcom-tool-kit/eslint))?
```

Comment:
**nitpick:** We should use a level 2 heading (`##`) as it directly proceeds a level 1 heading.